CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0] -
+### Updated
+- Upgraded plotly.js to 1.49.0 [#589](https://github.com/plotly/dash-core-components/pull/589)
+  - [Feature release 1.49.0](https://github.com/plotly/plotly.js/releases/tag/v1.49.0) which contains:
+
```

Comment:
@alexcjohnson Could you fill this in with the most significant items from the new release?

Reply:
OK, added. I'm not sure I need to always be the one to do this though, I might have a bit more context about this work but even I'm mostly just summarizing the biggest additions from the release notes.